atom/browser/api/atom_api_web_contents.cc

```diff
@@ -481,12 +481,8 @@ bool WebContents::DidAddMessageToConsole(content::WebContents* source,
                                          const base::string16& message,
                                          int32_t line_no,
                                          const base::string16& source_id) {
-  if (type_ == OFF_SCREEN) {
-    return false;
-  } else {
-    Emit("console-message", level, message, line_no, source_id);
-    return true;
-  }
+  Emit("console-message", level, message, line_no, source_id);
```

Comment:
Actually, can we `return Emit`

Reply:
That would semantically be correct as it returns false if the event isn't handled in JS land 👍